integration-tests/tests/src/hooks/open-realm-before.ts

```diff
@@ -54,12 +57,16 @@ export function openRealmHook(config: LocalConfiguration | SyncedConfiguration =
 export function closeRealm(this: RealmContext): void {
   if (this.realm) {
     this.realm.close();
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore Ignore error as we ensure the realm is reopened
```

Comment:
Perhaps `this` should be a `Partial<RealmContext>` instead?

Reply:
Yeah that would work!

Reply:
What was the ts error here?

Reply:
`The operand of a 'delete' operator must be optional` which is a new one to me!

Reply:
Ahhh...ok that makes sense.  Since `this` is typed with non-optionals.  So `Partial<RealmContext>` is appropriate